manualTests/DevicePropertiesGetAllAsync/Program.cs

```diff
@@ -0,0 +1,66 @@
+﻿// See https://github.com/vestervang/DotNet-BlueZ/issues/8
+
+using vestervang.DotNetBlueZ;
+// using HashtagChris.DotNetBlueZ;
+
+namespace DevicePropertiesGetAllAsync
+{
+    class Program
+    {
+        static async Task Main(string[] args)
+        {
+            using (var adapter = (await BlueZManager.GetAdaptersAsync()).FirstOrDefault())
```

Comment:
The same as in `DevicePropertiesGetAllAsync/Program.cs`

Reply:
Resolved on [cac7288](https://github.com/vestervang/DotNet-BlueZ/pull/10/commits/cac72886517e1d7aaf799dcdc3fd2d3f3135527c) (See comments above as well)